### DISCUSSION TITLE
Config setting to append SEED to output?

### DISCUSSION THREAD

**Discussion by setzke** [Sanster/IOPaint#231] [Q&A] [answered]
I didn't notice until several hours in of working that the program names any downloads the same name as the original, and there is no warning of overwrites... so if you make two edits to same picture and want both, you'll have to go to your output folder, rename the one you want to keep, then go save the new one.  Is there a way around this?

If not, can we get an option to append seed, prompt, etc to name, or in the very least allow for SAVE AS?

**Comment by Sanster** [comment, marked as answer]
You can remove --output-dir, the Save As dialog box will pop up each time you save

**Reply by setzke** [reply]
Thanks! Leaving input and output as "" in config will still use app directory, but removing those lines used browser default downloads folder.  Thanks a ton! Looking forward to the update to output folder.
